docs/content/install/shared-team-instance.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Shared Team Instance
+description: Installing an Athens Instance For Your Development Team
+---
+
+When you follow the instructions in the [Walkthrough](/walkthrough), you end up with an Athens Proxy that uses in-memory storage. This is only suitable for trying out the proxy for a short period of time, as you will quickly run out of memory and Athens won't persist modules between restarts. This guide will help you get Athens running in a more suitable manner for scenarios like providing an instance for your development team to share.
+
+We will use Docker to run the Athens proxy, so first make sure you have Docker [installed](https://docs.docker.com/install/).
+
+## Selecting a Storage Provider
+
+Athens currently supports a number of storage drivers. For local use we recommend starting with the local disk provider. For other providers, please see the Storage Provider documentation [Coming Soon].
+
+## Running Athens with Local Disk Storage
+
+In order to run Athens with disk storage, you will next need to identify where you would like to persist modules. In the example below, we will create a new directory named **athens-storage** in our current directory.  Now you are ready to run Athens with disk storage enabled. To enable disk storage, you need to set the **ATHENS_STORAGE_TYPE** and **ATHENS_DISK_STORAGE_ROOT** environment variables when you run the Docker container.
+
+**Bash**
+```bash
+export ATHENS_STORAGE=athens-storage
+mkdir $ATHENS_STORAGE
+docker run -d -v $ATHENS_STORAGE:/var/lib/athens \
+   -e ATHENS_DISK_STORAGE_ROOT=/var/lib/athens \
+   -e ATHENS_STORAGE_TYPE=disk \
+   --name athens-proxy \
+   --restart always \
+   -p 3000:3000 \
+   jeremyrickard/athens-proxy:latest
+```
+
+**PowerShell**
+```PowerShell
+set $env:ATHENS_STORAGE = "athens-storage"
+md -Path $env:ATHENS_STORAGE
+docker run -d -v $env:ATHENS_STORAGE:/var/lib/athens `
+   -e ATHENS_DISK_STORAGE_ROOT=/var/lib/athens `
+   -e ATHENS_STORAGE_TYPE=disk `
+   --name athens-proxy `
+   --restart always `
+   -p 3000:3000 `
+   jeremyrickard/athens-proxy:latest
+```
+
+Athens should now be running as a Docker container with the local directory, **athens-storage** mounted as a volume. When Athens retrieves the modules, they will be will be stored in the directory previously created. First, let's verify that Athens is running:
+
+```console
+$ docker ps | grep athens-proxy
+d658e9a211b9        jeremyrickard/athens-proxy:latest   "/bin/app"               About a minute ago   Up About a minute      0.0.0.0:3000->3000/tcp   athens-proxy
```

Comment:
This output has awkward lines breaks and we can't see the headers so it's kind of hard to understand what the output means. 

What do you think about not doing the grep and then showing truncated output where maybe we hide a few unhelpful columns like how long it's been up?